### PR TITLE
Schedule messages for winter 22/23 break

### LINKED ISSUE #40: Winter Break 22-23 alert
Schedule to appear: Mon. 12/19/22 through Tues. 1/3/23

Text: “The Stanford Libraries will be operating on a reduced schedule during the Stanford Winter Closure period (Wednesday, December 21, 2022 - Tuesday, January 3, 2023). More information about <a href=”https://library-hours.stanford.edu/”>hours</a>

Applications: SW, MyLibrary, Bento

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #40

### CODE REVIEW

File: sul.yaml [hunk 1]
```diff
@@ -1 +1,23 @@
-alerts: []
+---
+alerts:
+  - from: '2022-12-19T06:00:00-07:00'
+    to: '2023-01-03T22:00:00-07:00'
+    application_name: 'MyLibrary'
+    html: >
+      The Stanford Libraries will be operating on a reduced schedule during the Stanford Winter Closure
+      period (Wednesday, December 21, 2002 - Tuesday, January 3, 2023). More information about
```
Comment (corylown) [inline sul.yaml]:
```suggestion
      period (Wednesday, December 21, 2022 - Tuesday, January 3, 2023). More information about
```

File: sul.yaml [hunk 2]
```diff
@@ -1 +1,23 @@
...
+      <a href="https://library-hours.stanford.edu/">hours</a>
+  - from: '2022-12-19T06:00:00-07:00'
+    to: '2023-01-03T22:00:00-07:00'
+    application_name: 'SearchWorks'
+    html: >
+      The Stanford Libraries will be operating on a reduced schedule during the Stanford Winter Closure
+      period (Wednesday, December 21, 2002 - Tuesday, January 3, 2023). More information about
```
Comment (corylown) [inline sul.yaml]:
```suggestion
      period (Wednesday, December 21, 2022 - Tuesday, January 3, 2023). More information about
```

File: sul.yaml [hunk 3]
```diff
@@ -1 +1,23 @@
...
+      <a href="https://library-hours.stanford.edu/">hours</a>
+  - from: '2022-12-19T06:00:00-07:00'
+    to: '2023-01-03T22:00:00-07:00'
+    application_name: 'sul_bento_app'
+    html: >
+      The Stanford Libraries will be operating on a reduced schedule during the Stanford Winter Closure
+      period (Wednesday, December 21, 2002 - Tuesday, January 3, 2023). More information about
```
Comment (corylown) [inline sul.yaml]:
```suggestion
      period (Wednesday, December 21, 2022 - Tuesday, January 3, 2023). More information about
```

**Review by corylown — CHANGES_REQUESTED** [review]

File: sul.yaml (hunk 1 above)
Comment (jcoyne) [reply sul.yaml]: I will confirm with @saseestone. This is the message she asked for.  😆

Comment (corylown) [reply sul.yaml]: 2002 seems like yesterday, but...

**Review by corylown — APPROVED** [review]

**Pull request merged**